### ISSUE TITLE
Offline Height of Plot no respected in SageMaker

### ISSUE THREAD

**Issue by kbrostrom** [plotly/plotly.py#1572]
I'm trying to use plotly offline iplot in JupyterLab on AWS SageMaker. When I use offline iplot, the graph adjusts to the default cell output heigh of 360px instead of adjust the cell output size to the default graph height of 450px. When I try to set the height to a fixed height of 600px, the cell doesn't adjust and cuts off the bottom half of the graph.

I followed the instructions for installing all necessary widgets and extensions to work in JupyterLab. The following is my code.

`import plotly.offline as off
import plotly.graph_objs as go
import numpy as np

off.init_notebook_mode(connected=True)

trace = go.Heatmap(z=[[1, 20, 30, 50, 1], [20, 1, 60, 80, 30], [30, 60, 1, -10, 20]],
                   x=['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'],
                   y=['Morning', 'Afternoon', 'Evening'])
data=[trace]
layout = go.Layout(title='Activity Heatmap')

f2 = go.Figure(data,layout)

# two different ways to view f2
off.iplot(f2)
`
Here is a picture of my output

![image](https://user-images.githubusercontent.com/47127806/58195579-23822a00-7c7d-11e9-836e-838042d401ed.png)

When I try to set the height manually to 600px in the layout, here is what I'm returned:

![image](https://user-images.githubusercontent.com/47127806/58195634-48769d00-7c7d-11e9-8525-ea44c6e852a5.png)
 
I've tried with both go.Figure and go.FigureWidget

**Comment by jonmmease** [comment]
Hi @kbrostrom,

Thanks for the report. I think this may be the same issue that's being addressed in https://github.com/plotly/plotly.py/pull/1571.

Would you be able to try out the dev build to see if it fixes the problem for you?  You can install the dev build using pip with
```
$ pip install https://12899-14579099-gh.circle-artifacts.com/0/dist/plotly-3.9.0%2B4.g576a8b10.tar.gz
```

Thanks!

**Comment by jonmmease** [comment]
Fixed in https://github.com/plotly/plotly.py/pull/1571 and will be released in 3.10.0
